libraries/Microsoft.Bot.Builder.AI.QnA/Models/QnAMakerTraceInfo.cs

```diff
@@ -60,13 +60,14 @@ public class QnAMakerTraceInfo
 
         /// <summary>
         /// Gets or sets the filters used to return answers that have the specified metadata.
+        /// [Obsolete("This field is no longer used")].
```

Comment:
This attribute should be directly on the property. It's great to *also* say this in the doc, but it really needs to be on both. 

You can see some [examples of this in our C# sdk](https://github.com/microsoft/botbuilder-dotnet/blob/43fd9e605009794ff78b68b38e06b360a4617ebe/libraries/Microsoft.Bot.Builder.Dialogs/Memory/ScopePath.cs#L47).

Reply:
Made changes as suggested.